src-ui/lib/components/BooksTableRow.svelte

```diff
@@ -3,17 +3,33 @@
 
   export let book: Book
   export let handleEdit: (book: Book, field: keyof Book, event: Event) => void
+
+  const handleEnter = () => (event: Event) => {
+    if (event instanceof KeyboardEvent && event.key === 'Enter') {
+      event.preventDefault?.()
+      ;(event.currentTarget as HTMLTableCellElement).blur?.()
+    }
+  }
 </script>
 
 <tr>
-  <td contenteditable="true" on:blur={(e) => handleEdit(book, 'isbn10', e)}
-    >{book.isbn10 ?? book.isbn13}</td
+  <td
+    contenteditable="true"
+    on:blur={(e) => handleEdit(book, 'isbn10', e)}
+    on:keydown={handleEnter()}>{book.isbn10 ?? book.isbn13}</td
```

Comment:
aside: we should probably figure out a way to make editing the isbn save in the correct field -- or allow displaying / editing both.